test/unit/test_unified_utils.py

```diff
@@ -7,22 +8,27 @@
 
 TEXT_COL = "sentence1"
 
+
 class TestUnifiedExplainerUtils(object):
-    
+
     def test_working(self):
         assert True
 
     def test_get_single_embedding(self):
         model = get_bert_model()
-        device = torch.device("cpu" if not torch.cuda.is_available() else "cuda")
+        device = torch.device(
+            "cpu" if not torch.cuda.is_available() else "cuda")
         text = "rare bird has more than enough charm to make it memorable."
         embedded_input = _get_single_embedding(model, text, device)
         assert embedded_input is not None
 
+    @pytest.mark.xfail(reason="Unknown")
```

Comment:
Have we made Janhavi/Eddie aware of this?

Reply:
ok never mind, I see the comments on teams

Reply:
However, it shouldn't be there. @janhavi13  fixed it, so my merge from master obviously went wrong.

Reply:
Have brought in the version from master.